packages/e2e-test-app/jest.config.js

```diff
@@ -26,9 +33,11 @@ module.exports = {
 
   // A map from regular expressions to paths to transformers
   transform: {
-    '^.+\\.(bmp|gif|jpg|jpeg|mp4|png|psd|svg|webp)$':
-      'react-native-windows/jest/assetFileTransformer.js',
-    '.*': 'react-native-windows/jest/preprocessor.js',
+    '^.+\\.(bmp|gif|jpg|jpeg|mp4|png|psd|svg|webp)$': assetTransform,
+    'node_modules\\\\@?react-native\\\\.*': reactNativeTransform,
+    '@react-native-windows\\\\tester\\\\.*': reactNativeTransform,
+    'vnext\\\\.*': reactNativeTransform,
+    '^.+\\.[jt]sx?$': defaultTransform,
```

Comment:
e2e-test-app is special in that it loads the RN environment so it can load RNTester, including the Flow bits. I was having it use React Native's Babel transform to do this, but the transform kept on hitting issues in TypeScript code indeterminately, including in this PR (seems stateful).

I changed to only use the RN transform in Flow packages, and use the default transform everywhere else.